sylt-common/src/ty.rs

```diff
@@ -214,6 +214,14 @@ impl Type {
                         ));
                     }
                 }
+                if a_args.len() != b_args.len() {
+                    return Err(
+                        format!(
+                            "mismatching arity, {} != {}",
```

Comment:
```suggestion
                            "Mismatching arity, {} != {}",
```

Reply:
This is already uniform with other errors